lib/core_modules/module/dependablelist.js

```diff
@@ -51,7 +51,7 @@ Module.prototype.path = function(extra){
 };
 Module.prototype.activate = function(){
   var req = require(this.path('app.js'));
-  if(req && 'function' === req.init){
+  if(req && 'function' === typeof(req.init)){
```

Comment:
While I agree that the existing code is incorrect, `typeof` is not itself a function. So you should drop the parentheses. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof

```
if(req && 'function' === typeof req.init){
```